nebula_ros/include/nebula_ros/continental/continental_ars548_ros_wrapper.hpp

```diff
@@ -0,0 +1,102 @@
+// Copyright 2024 Tier IV, Inc.
```

Comment:
TIER IV :police_car:

Reply:
Thanks !
Fixed in [2a1e08b](https://github.com/tier4/nebula/pull/151/commits/2a1e08b92b4a4dfbde88094479475ce56d85d5ef)